install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,124 +11,248 @@
+claims (PVCs), however, are specific to a project and are created and used by developers.
+
+(*Note:* "project" and "namespace" are used interchangeably throughout this document)
 
 == Provisioning
-Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision NFS volumes in OpenShift, all that is
-required is a distinct list of NFS servers and paths and the
-`*PersistentVolume*` API.
 
-You must define your persistent volume in an object definition before creating
-it in OpenShift:
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in
+OpenShift. To provision NFS volumes a list of NFS servers and export paths are all that is required.
 
-.Persistent Volume Object Definition Using NFS
-====
+*Note:*
+Each NFS volume must be mountable by all schedulable nodes in the cluster.
 
-[source,yaml]
-----
-apiVersion: "v1"
-kind: "PersistentVolume"
+A PV spec file is usually defined before creating the PV:
+```
+apiVersion: v1
+kind: PersistentVolume
 metadata:
-  name: "pv0001" <1>
+  name: pv0001 <1>
 spec:
   capacity:
-    storage: "5Gi" <2>
+    storage: 5Gi <2>
   accessModes:
-    - "ReadWriteOnce" <3>
+  - ReadWriteOnce <3>
   nfs: <4>
-    path: "/tmp" <5>
-    server: "172.17.0.2" <6>
-  persistentVolumeReclaimPolicy: "Recycle" <7>
-----
-<1> The name of the volume. This will be how it is identified via
-link:../../architecture/additional_concepts/storage.html[persistent volume
-claims] or from pods.
-<2> The amount of storage allocated to this volume.
-<3> This is the policy for restricting read/writes to a shared volume.
-<4> This defines the volume type being used, in this case the *nfs* plug-in.
-<5> The path that is exported by the NFS server.
-<6> The host name or IP address of the NFS server.
-<7> Defines what happens to a volume when released from its claim. Valid options
+    path: /tmp <5>
+    server: 172.17.0.2 <6>
+  persistentVolumeReclaimPolicy: Recycle <7>
+```
+<1> the name of the volume. This is the PV identify in various `oc ... pod` commands.
+<2> the amount of storage allocated to this volume.
+<3> though this appears to be related to controlling access to the volume, in fact, it
+is actually used similarly to labels and used to match a PVC to a PV. Currently, no
+access rules are enforced based on the `accessModes`.
+<4> this defines the volume type being used, in this case the *nfs* plug-in.
+<5> the path that is exported by the NFS server.
+<6> the host name or IP address of the NFS server.
+<7> defines what happens to a volume when released from its claim. Valid options
 are *Retain* (default) and *Recycle*. See
 link:#nfs-reclaiming-resources[Reclaiming Resources].
-====
 
-Save your definition to a file, for example *_nfs-pv.yaml_*, and create the
-persistent volume:
-
-====
-----
+Save the definition to a file, for example *_nfs-pv.yaml_*, and create the persistent
+volume:
+```
```

Comment:
Switch back to former block formatting. I'll stop commenting to this effect; applies to the rest of the changes.